src/rhsmlib/file_monitor.py

```diff
@@ -172,15 +178,15 @@ def __init__(self, path, callbacks, is_glob=False):
         :param callbacks: list of methods called when directory is changed
         :param is_glob: bool - if path provided is glob or not
         """
-        self.IN_CREATE = 0x00000100
         self.IN_DELETE = 0x00000200
         self.IN_MODIFY = 0x00000002
 
-        self.path = path
+        self.path = os.path.abspath(path)
+        self.is_file = os.path.isfile(self.path)
         self.timestamp = None
         self.is_glob = is_glob
         self.callbacks = callbacks
-        self.mask = self.IN_CREATE | self.IN_DELETE | self.IN_MODIFY
+        self.mask = self.IN_DELETE | self.IN_MODIFY
```

Comment:
Presuming the removal of IN_CREATE is to reduce the duplicate signals being fired on creation of a file?

Reply:
Yes, since upon only creation of a new file, both IN_MODIFY and IN_CREATE are fired, I figured IN_CREATE was kinda redundant